hive/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -184,6 +183,9 @@ public void commit(TableMetadata base, TableMetadata metadata) {
         });
       }
       threw = false;
+    } catch (org.apache.hadoop.hive.metastore.api.AlreadyExistsException e) {
```

Comment:
This part requires attention. This change was needed to always throw `AlreadyExistsException` instead of `RuntimeException` when tables are created concurrently.